src/xcode/ENA/ENA/Source/Services/Exposure Transaction/ExposureDetection.swift

```diff
@@ -22,7 +22,6 @@ import Foundation
 final class ExposureDetection {
 
 	// MARK: Properties
-	@Published var activityState: RiskProvider.ActivityState = .idle
```

Comment:
Sad to see this being removed…

Reply:
Why are you sad? The ExposureDetection has no different states anymore. Why do we need it?

Reply:
Must have been confused with the `.idle` in `ExposureDetectionViewController+State` then.